firstvoices/backend/tasks/send_email_tasks.py

```diff
@@ -0,0 +1,31 @@
+from smtplib import SMTPException
+
+from celery import shared_task
+from django.conf import settings
+from django.core.mail import send_mail
+
+from backend.views.exceptions import ContactUsError
+
+
+@shared_task
+def send_contact_us_email(name, from_email, message, to_email_list):
```

Comment:
What if we made this a generic send_email task, and had the contact_us view pass in the formatted message and subject? It would be useful for sending other notifications, and I think it would also be nice to be able to read the full message and subject info right in the contact_us_view, for troubleshooting

Reply:
No need for a from_email parameter here, I think

Reply:
Great suggestion, thanks. I have made the task reusable and updated the tests.